shared/templates/package_installed/tests/package-installed-removed.fail.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+{{{ bash_package_install(PKGNAME) }}}
```

Comment:
If the comment about redundancy was here, note that Debian systems can differentiate between a package that has never been installed versus one that has been installed before. We want to make sure that the OVAL correctly detects that the package is still removed and doesn't e.g., behave incorrectly on packages that were previously installed.

Note that we actually had this bug in the package installed bash check before: `dpkg -L | grep <name>` is NOT sufficient of a check for whether a package is present because removed packages will show up in that list. That's how we ended up with the more [complex query presently](https://github.com/ComplianceAsCode/content/blob/master/shared/macros-bash.jinja#L675):

```bash

{{%- macro bash_package_installed(pkgname) -%}}
{{%- if pkg_manager == "apt_get" -%}}
dpkg-query --show --showformat='${db:Status-Status}\n' "{{{ pkgname }}}" 2>/dev/null | grep -q installed
{{%- else -%}}
rpm --quiet -q "{{{ pkgname }}}"
{{%- endif -%}}
{{%- endmacro -%}}
```